server/server.js

```diff
@@ -1,60 +1,64 @@
-const express = require('express')
-const bodyParser = require('body-parser')
-const session = require('express-session')
-const app = express()
-const axios = require("axios")
-require('dotenv').config();
+const express = require("express");
```

Comment:
All linter changes except adding an `isDev` param on line 25 and logic necessary to support that